### ISSUE TITLE
Version check throws error

### ISSUE THREAD

**Issue by George9000** [codota/tabnine-nvim#85]
Error with latest change in `tabnine.lua` checking for version:
```
Error detected while processing /Users/foo/.config/nvim/init.lua:
E5113: Error while calling lua chunk: ...foo/.local/share/nvim/lazy/tabnine-nvim/lua/tabnine.lua:14: attempt to index field 'version' (a function value)
stack traceback:
        ...foo/.local/share/nvim/lazy/tabnine-nvim/lua/tabnine.lua:14: in function 'setup'
```

**Comment by aarondill** [comment]
@amirbilu, it appears I missed that vim.version was expanded in version `0.9.0` to include `vim.version.*` rather than just `vim.version()`

**Comment by George9000** [comment]
```
NVIM v0.9.0-dev
Build type: Release
LuaJIT 2.1.0-beta3

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/opt/local/share/nvim"
```

**Comment by aarondill** [comment]
huh, you're running `0.9.0`? what does `test -f /usr/share/nvim/runtime/lua/vim/version.lua && echo yes || echo no` result in?

**Comment by aarondill** [comment]
looks like you have a different installation directory than me, try `test -f /opt/local/share/nvim/runtime/lua/vim/version.lua && echo yes || echo no` instead

**Comment by aarondill** [comment]
It's interesting that you are actually running into this problem (though I'm glad you pointed it out), because the nvim tag `v0.9.0` includes a [version.lua](https://github.com/neovim/neovim/blob/v0.9.0/runtime/lua/vim/version.lua) file that has the methods we're looking for.

**Comment by aarondill** [comment]
hopefully the latest version fixes this, but I'm still curious as to whether your installation includes the version.lua file :smile:

**Comment by amirbilu** [comment]
I hope you test everything cause I'm in limited availability:)

**Comment by George9000** [comment]
BTW, new version of this plugin loaded without error.

Output of above command: 
```
% ls -la /opt/local/share/nvim/runtime/lua/vim/                           14:27
total 272
drwxr-xr-x  23 root      wheel    736 Jun 10 21:52 .
drwxr-xr-x   8 root      wheel    256 Jun 10 21:52 ..
-rw-r--r--   1 macports  wheel    844 Feb 23 03:15 F.lua
-rw-r--r--   1 macports  wheel  28505 Feb 23 03:15 _editor.lua
-rw-r--r--   1 macports  wheel   2806 Feb 23 03:15 _init_packages.lua
-rw-r--r--   1 macports  wheel   7824 Feb 23 03:15 _inspector.lua
-rw-r--r--   1 macports  wheel  12347 Feb 23 03:15 _meta.lua
-rw-r--r--   1 macports  wheel  57609 Feb 23 03:15 diagnostic.lua
drwxr-xr-x   3 root      wheel     96 Jun 10 21:52 filetype
-rw-r--r--   1 macports  wheel  74989 Feb 23 03:15 filetype.lua
-rw-r--r--   1 macports  wheel   9406 Feb 23 03:15 fs.lua
-rw-r--r--   1 macports  wheel   1574 Feb 23 03:15 health.lua
-rw-r--r--   1 macports  wheel   4429 Feb 23 03:15 highlight.lua
-rw-r--r--   1 macports  wheel   9077 Feb 23 03:15 inspect.lua
-rw-r--r--   1 macports  wheel   3659 Feb 23 03:15 keymap.lua
drwxr-xr-x  15 root      wheel    480 Jun 10 21:52 lsp
-rw-r--r--   1 macports  wheel  83858 Feb 23 03:15 lsp.lua
-rw-r--r--   1 macports  wheel   4946 Feb 23 03:15 secure.lua
-rw-r--r--   1 macports  wheel  21823 Feb 23 03:15 shared.lua
drwxr-xr-x   9 root      wheel    288 Jun 10 21:52 treesitter
-rw-r--r--   1 macports  wheel  17762 Feb 23 03:15 treesitter.lua
-rw-r--r--   1 macports  wheel   3760 Feb 23 03:15 ui.lua
-rw-r--r--   1 macports  wheel   3455 Feb 23 03:15 uri.lua
```

**Comment by aarondill** [comment]
seems odd to me that you're apparently missing a few files considering neovim is reporting it's version as v0.9.0, but I'm glad the problem was resolved (since we still support >=0.7.1 anyways)
